recipes/dbg-macro/all/conanfile.py

```diff
@@ -21,6 +22,19 @@ def source(self):
         extracted_dir = self.name + "-" + self.version
         os.rename(extracted_dir, self._source_subfolder)
 
+    def configure(self):
+        minimal_cpp_standard = "11"
+        if self.settings.get_safe("compiler.cppstd"):
+            tools.check_min_cppstd(self, minimal_cpp_standard)
+
+        if tools.Version(self.settings.compiler.version) < "5":
```

Comment:
What happens on apple-clang and msvc here ?

Reply:
True, but apple-clang is hard to use old versions, as OSX imposes new versions and msvc people usually run 2015 or higher. GCC is another history, people still use old versions for embedded stuff for instance. Anyway, I'm not opposed to adding both.

Reply:
Lost gcc if during my local debug on mac with apple-clang.
Added it back.
Good note, @ericLemanissier !